src/styles/getTooltipStyles.js

```diff
@@ -67,37 +67,31 @@ function computeHorizontalShift(windowWidth, xOffset, componentWidth, tooltipWid
  *                           and the top edge of the wrapped component.
  * @param {Element} [tooltip] - The reference to the tooltip's root element
  * @param {Number} tooltipTargetWidth - The width of the tooltip's target
- * @param {Number} tooltipTargetHeight - The height of the tooltip's target
  * @returns {Boolean}
  */
-function isOverlappingAtTop(xOffset, yOffset, tooltip, tooltipTargetWidth, tooltipTargetHeight) {
+function isOverlappingAtTop(xOffset, yOffset, tooltip, tooltipTargetWidth) {
     if (typeof document.elementFromPoint !== 'function') {
         return false;
     }
 
-    // Use the x and y center position of the target to prevent wrong element
-    // returned by elementFromPoint in case the target has a border radius or is a multiline text.
+    // Use the x center position of the target to prevent wrong element returned by elementFromPoint
+    // in case the target has a border radius or is a multiline text.
     const targetCenterX = xOffset + tooltipTargetWidth / 2;
-    const targetCenterY = yOffset + tooltipTargetHeight / 2;
     const elementAtTargetCenterX = document.elementFromPoint(targetCenterX, yOffset);
-    const elementAtTargetCenterY = document.elementFromPoint(xOffset, targetCenterY);
     const tooltipRef = (tooltip && tooltip.current) || tooltip;
 
     // Ensure it's not the already rendered element of this very tooltip, so the tooltip doesn't try to "avoid" itself
-    if (!elementAtTargetCenterX || !elementAtTargetCenterY || (tooltipRef && (tooltipRef.contains(elementAtTargetCenterX) || tooltipRef.contains(elementAtTargetCenterY)))) {
+    if (!elementAtTargetCenterX || (tooltipRef && tooltipRef.contains(elementAtTargetCenterX))) {
         return false;
     }
 
     const rectAtTargetCenterX = elementAtTargetCenterX.getBoundingClientRect();
-    const rectAtTargetCenterY = elementAtTargetCenterY.getBoundingClientRect();
 
     // Ensure it's not overlapping with another element by checking if the yOffset is greater than the top of the element
     // and less than the bottom of the element
     const isOverlappingAtTargetCenterX = yOffset > rectAtTargetCenterX.top && yOffset < rectAtTargetCenterX.bottom;
-    const isOverlappingAtTargetCenterY = yOffset > rectAtTargetCenterY.top && yOffset < rectAtTargetCenterY.bottom;
 
-    // Return true only if both elementAtTargetCenterX and elementAtTargetCenterY overlap with another element
-    return isOverlappingAtTargetCenterX && isOverlappingAtTargetCenterY;
+    return isOverlappingAtTargetCenterX;
```

Comment:
This caused a regression https://github.com/Expensify/App/issues/26041. Now we not taking into consideration the height the tooltip shows up at the bottom if there is any element above the tooltip marker, e.g. if we have the unread indicator shown before a report action then the tooltip will show at bottom even though we have enough space at the top.

Before:
<img width="1840" alt="Screenshot 2023-09-09 at 11 01 04 PM" src="https://github.com/Expensify/App/assets/16493223/bc06f304-71c8-47ae-a116-57f6c625c4a0">

After:
<img width="1840" alt="Screenshot 2023-09-09 at 11 01 23 PM" src="https://github.com/Expensify/App/assets/16493223/f89186d4-7c0a-4ee5-b400-c23214d81106">

Reply:
The issue that this PR is fixing does not seem to be reproducible, do you think we can revert the PR?

Reply:
Thanks for the quick response. I think it would be safer to just ignore the unread indicator and keep the overlapping logic as is for now.